pkg/scm/git/git.go

```diff
@@ -34,7 +34,7 @@ type stiGit struct {
 	runner util.CommandRunner
 }
 
-var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/]+\.git)$`)
+var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/])`)
```

Comment:
But the point of that particular test was to allow URLs without .git at the end.  If I add $, then that URL without .git fails.